R/plotImage.R

```diff
@@ -49,9 +55,12 @@ plotSpatialData <- \() ggplot() + scale_y_reverse() + .theme
         c <- col2rgb(c)/255
         b <- array(0, dim=c(3, dim(a)[-1]))
         for (i in seq_len(dim(a)[1])) {
-            b[1,,] <- b[1,,,drop=FALSE] + a[i,,,drop=FALSE]*c[1,i]
-            b[2,,] <- b[2,,,drop=FALSE] + a[i,,,drop=FALSE]*c[2,i]
-            b[3,,] <- b[3,,,drop=FALSE] + a[i,,,drop=FALSE]*c[3,i]
+            b[1,,] <- b[1,,,drop=FALSE] + a[i,,,drop=FALSE]*c[1,i]*(1/lim[[i]][2])*sat[i]
```

Comment:
I am actually not so sure about the `sat` part... seems redundant as is, so maybe it should be applied differently? (1/lim) makes sense to me and is also what I found in imaging literature for how contrast limits work.